tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist=py{26,27,33,34}-django16, py{27,33,34}-django17
+envlist=py{26,27,33,34}-django16-cms{30,31,32'}, py{27,33,34}-django{17,18}-cms{31,32}
```

Comment:
I'd have written this like:
````
py{34,33,27}-django{18,17}-cms{32,31}, py{34,33,27,26}-django16-cms{32,31,30}
````

With this, you can (mostly) generate your travis env-list with `tox -l`

Also, you have a typo. There's a single quote in `cms{30, 31, 32'}`